### DISCUSSION TITLE
How to change CSS padding/height for images and logos ?!

### DISCUSSION THREAD

**Discussion by OmarGhoche** [Budibase/budibase#5355] [Help] [answered]
I am having problems to display my tables ... the logo is big and passes the menu limit ... i tried to shrink it using custom css (I found out the class to be something like `.logo.svelte-1qctuec img.svelte-1qctuec` and tried to reduce the height) but did'nt work. Nor the side or lower padding/margin (css class `.size--s.svelte-1qctuec.svelte-1qctuec`) of the tables.

![](https://raw.githubusercontent.com/pontus-vision/README-images/main/bb-borders.png)

How could I make these changes globally so I end up with a layout similar to this 👇

![](https://raw.githubusercontent.com/pontus-vision/README-images/main/bb-border-2.png)

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hey @OmarGhoche 

The custom CSS is inline only, therefore you cannot use class selectors. Any CSS is simply applied to the selected component.

I'm assuming there's no way for you to get a link to a smaller version of the logo? 

There is a feature request here that looks to offer more flexibility with handling / storing images: https://github.com/Budibase/budibase/issues/5285

**Reply by OmarGhoche** [reply]
Got it @melohagan ... thanks for the clarification!

**Comment by tw02d008** [comment]
I have the same issue with the logo an the navigation bar.
smaller version of the logo doesn't help either because the smaller one will be resized to the bigger size.
Looks like a bug?
